app/controllers/issue_templates_controller.rb

```diff
@@ -1,206 +1,219 @@
+    if inherit_template
+      # keep ordering
+      used_tracker_ids = @project.trackers.pluck(:tracker_id)
+=begin
+      project_ids.each do |i|
+        @inherit_templates.concat(
+            IssueTemplate.search_by_project(i)
+                .search_by_tracker(used_tracker_ids)
+                .enabled
+                .enabled_sharing
+                .order_by_position
+        )
+      end
+=end
+      @inherit_templates = get_inherit_templates(project_ids, used_tracker_ids)
+    end
+
+    @global_issue_templates = GlobalIssueTemplate.joins(:projects)
+                                 .search_by_project(@project.id)
+                                 .order_by_position
+
+
+    render layout: !request.xhr?
+  end
+
+  def show
+  end
+
+  def new
+    # create empty instance
+    @issue_template ||= IssueTemplate.new(author: @user, project: @project)
+    if request.post?
+      @issue_template.safe_attributes = params[:issue_template]
+      if @issue_template.save
+        flash[:notice] = l(:notice_successful_create)
+        redirect_to action: 'show', id: @issue_template.id, project_id: @project
+      end
+    end
+  end
+
+  def edit
+    # Change from request.post to request.patch for Rails4.
+    if request.patch? || request.put?
+      @issue_template.safe_attributes = params[:issue_template]
+      if @issue_template.save
+        flash[:notice] = l(:notice_successful_update)
+        redirect_to action: 'show', id: @issue_template.id, project_id: @project
+      end
+    end
+  end
+
+  def destroy
+    if request.post?
+      if @issue_template.destroy
+        flash[:notice] = l(:notice_successful_delete)
+        redirect_to action: 'index', project_id: @project
+      end
+    end
+  end
+
+  # load template description
+  def load
+    @issue_template = if !params[:template_type].nil? && params[:template_type] == 'global'
+                        GlobalIssueTemplate.find(params[:issue_template])
+                      else
+                        IssueTemplate.find(params[:issue_template])
+                      end
+    render text: @issue_template.to_json(root: true)
+  end
+
+  # update pulldown
+  def set_pulldown
+    grouped_options = []
+    group = []
+    default_template = nil
+    setting = IssueTemplateSetting.find_or_create(@project.id)
+    inherit_template = setting.enabled_inherit_templates?
+
+    project_ids = inherit_template ? @project.ancestors.collect(&:id) : [@project.id]
+    issue_templates = IssueTemplate.search_by_project(@project.id)
+                          .search_by_tracker(@tracker.id)
+                          .enabled.order_by_position
+
+    project_default_template = issue_templates.is_default.first
+
+    unless project_default_template.blank?
+      default_template = project_default_template.id
+    end
+
+    unless issue_templates.empty?
+      issue_templates.each { |x| group.push([x.title, x.id]) }
+    end
+
+    if inherit_template
+=begin
```

Comment:
__[rubocop]__
* Do not use block comments. [:ref](https://github.com/bbatsov/ruby-style-guide#no-block-comments)